resources/nspawn.go

```diff
@@ -115,7 +172,7 @@ func (obj *NspawnRes) Watch() error {
 	if err := call.Err; err != nil {
 		return err
 	}
-	buschan := make(chan *dbus.Signal, 10)
```

Comment:
I'm not sure about this one. Are you?
At minimum add a TODO to look into this more (whether we give the channel a buffer or not)
Try repeatedly sending signals to see if it drops one without a buffer.
Eg: https://github.com/purpleidea/mgmt/blob/master/resources/packagekit/packagekit.go#L42

(I think eventually someone, hopefully not us, should dig into the implementation to understand why it does that and if there is a fix...)